gradboost_pv/inference/models.py

```diff
@@ -483,20 +487,26 @@ def predict_from_covariates(self, covariates: Covariates) -> Dict[Hour, Predicti
     def process_model_output(
         self,
         forecast_horizon_hours: Hour,
-        forecast: float,
+        forecast: np.ndarray,
         pv_capacity_mwp: float,
         inference_datetime: np.datetime64,
+        lower_scaling: float = 1.0,
+        upper_scaling: float = 1.0,
```

Comment:
could you add a description in the function descritpion, bonus if you can add a quick description of the other inputs too